deployment/helm/ditto/values.yaml

```diff
@@ -0,0 +1,1572 @@
+      endpoint: ""
+    # logFiles defines logging to log files config
+    logFiles:
+      # enabled whether to write logs to log files
+      #  log files can be found on the host under /var/log/ditto
+      enabled: false
+    # customConfigFile configures that a custom "Logback" config file should be used instead of the one bundled
+    #  with Ditto on the classpath
+    customConfigFile:
+      # enabled if enabled, a custom logback.xml file added to the Ditto containers will be used for logging configuration
+      enabled: true
+      # fileName passed as Java system property "-Dlogback.configurationFile"
+      fileName: logback.xml
+  # tracing configuration for Ditto
+  tracing:
+    # enabled whether tracing (via OpenTelemetry) is enabled
+    enabled: false
+    # otelExporterOtlpEndpoint the OTLP endpoint to report traces to
+    otelExporterOtlpEndpoint: "http://localhost:4317"
+    # sampler the tracing sampler to use
+    #  can be one of:
+    #   - always: report all traces.
+    #   - never:  don't report any trace.
+    #   - random: randomly decide using the probability defined in the random-sampler.probability setting.
+    #   - adaptive: keeps dynamic samplers for each operation while trying to achieve a set throughput goal.
+    sampler: never
+    # randomSampler configures the 'random' sampler
+    randomSampler:
+      # probability configures the probability of a span being sampled, must be a value between 0 and 1
+      probability: 0.01
+    # adaptiveSampler configures the 'adaptive' sampler
+    adaptiveSampler:
+      # throughput the throughput goal trying to achieve with the adaptive sampler
+      throughput: 600
+
+## ----------------------------------------------------------------------------
+## dbconfig for mongodb connections
+##  will be handled as k8s secret as connection uri might contain auth credentials
+dbconfig:
+  # policies the MongoDB configuration for Ditto "policies" service
+  policies:
+    uri: mongodb://#{PLACEHOLDER_MONGODB_HOSTNAME}#:27017/ditto
+    ssl: false
+  # things the MongoDB configuration for Ditto "things" service
+  things:
+    uri: mongodb://#{PLACEHOLDER_MONGODB_HOSTNAME}#:27017/ditto
+    ssl: false
+  # connectivity the MongoDB configuration for Ditto "connectivity" service
+  connectivity:
+    uri: mongodb://#{PLACEHOLDER_MONGODB_HOSTNAME}#:27017/ditto
+    ssl: false
+  # thingsSearch the MongoDB configuration for Ditto "things-search" service
+  thingsSearch:
+    uri: mongodb://#{PLACEHOLDER_MONGODB_HOSTNAME}#:27017/ditto
+    ssl: false
+  ## If following property is set, an existing secret will be used to retrieve the mongodb connectionUris from.
+  # uriSecret: my-uri-secret
+
+## ----------------------------------------------------------------------------
+## ingress configures the Ingress
+ingress:
+  # enabled whether Ingress should be enabled as alternative to the contained nginx
+  enabled: false
```

Comment:
if this is set to true shouldn't we also have a deployment for the respecitve nginx-ingress-controller.
Currently as it is setup it would just search for the nginx-pod

Reply:
Oh, true..
But probably with another flag, as there might be environments where there already is an ingress controller in place.

Do you have experience in that (Ingress controller)? I don't.. 😕

Reply:
From what I see we need to create a deployment for this.
Maybe I can try it manually on my end -> create an example template file -> and you can add it to the PR ?

Reply:
Sounds good

Reply:
just as an update currently still working on it :)
I can if you want push the files that I added.
Currently struggling with a permissions issue inside the ingress pod that I am creating.

Reply:
> just as an update currently still working on it :)
> I can if you want push the files that I added.

That would be cool.
I however think you have no permissions to push to a Ditto branch yet.. 😉

Reply:
@thjaeckle 
Yes, that is a good idea. 
Maybe we should do that and we can add the PR separately.